src/services/uiLogger.ts

```diff
@@ -71,6 +71,8 @@ const createData = (message?: string, error?: Error, additionalData?: Object): I
 };
 
 const log = (message: string, error?: Error, additionalData?: Object) => {
+  if (window.location.hostname === 'localhost') return;
```

Comment:
For local development we still need some logs to be visible, so that we are aware there is some issue